pkg/status/status.go

```diff
@@ -0,0 +1,21 @@
+package status
+
+type Stage int
+type State int
+
+const (
+	Configure Stage = iota
+	Build
+)
+
+const (
+	Failed State = iota
+	InProgress
+	Success
+)
+
+type BuildStatus struct{
+	BuildStage 	Stage
+	BuildStatus State
+	Message 	string
+}
```

Comment:
1. Why do we need a struct for Build? If it is just a string which is sent as a message why not just construct a string with type restraints
2. Can we make the build and configure stage message structure as similar as possible?